pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/PicklistQuestionDao.java

```diff
@@ -35,7 +35,7 @@ public interface PicklistQuestionDao extends SqlObject {
 
     Logger LOG = LoggerFactory.getLogger(PicklistQuestionDao.class);
 
-    int DISPLAY_ORDER_GAP = 10;
+    public final int DISPLAY_ORDER_GAP = 10;
```

Comment:
I believe it is public by default according to the [specification](https://docs.oracle.com/javase/specs/jls/se7/html/jls-6.html#jls-6.6.1)

Reply:
Yup- they are. Given these SqlObjects sit in a weird space, these all were colored as `package-private`. Given it's unnecessary, I'll go through and remove them to match the style in the rest of the definition.